<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #fffff0;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="2">
    <widget class="QPushButton" name="pushButton_exit">
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #ffffff;
    color: #333333;
    border: 2px solid #0078d7;         /* синий акцент */
    border-radius: 8px;
    padding: 6px 12px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #e6f0ff;         /* светло-синий при наведении */
}

QPushButton:pressed {
    background-color: #cce0ff;         /* темнее при нажатии */
}

QPushButton:disabled {
    background-color: #f0f0f0;
    color: #a0a0a0;
    border: 1px solid #cccccc;
}
#pushButton_duplicate {
    background-color: #f9f9f9;
    border: 2px solid #28a745;   /* зелёный */
    border-radius: 8px;
    padding: 6px 14px;
}
#pushButton_duplicate:hover {
    background-color: #e6ffe9;
}
</string>
     </property>
     <property name="text">
      <string>Выход</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: #5c4033;          /* тёмно-серый текст */
    font-size: 14px;
    font-weight: 500;
    padding: 2px 4px;
}
</string>
     </property>
     <property name="text">
      <string>Выберите причину отказа от подхода</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pushButton_ok">
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #ffffff;
    color: #333333;
    border: 2px solid #0078d7;         /* синий акцент */
    border-radius: 8px;
    padding: 6px 12px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #e6f0ff;         /* светло-синий при наведении */
}

QPushButton:pressed {
    background-color: #cce0ff;         /* темнее при нажатии */
}

QPushButton:disabled {
    background-color: #f0f0f0;
    color: #a0a0a0;
    border: 1px solid #cccccc;
}
#pushButton_duplicate {
    background-color: #f9f9f9;
    border: 2px solid #28a745;   /* зелёный */
    border-radius: 8px;
    padding: 6px 14px;
}
#pushButton_duplicate:hover {
    background-color: #e6ffe9;
}
</string>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_method_rejection">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: #5c4033;          /* тёмно-серый текст */
    font-size: 17px;
    font-weight: 500;
    padding: 2px 4px;
}
</string>
     </property>
     <property name="text">
      <string>Отказ от ... подхода</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QComboBox" name="comboBox_reason">
     <property name="styleSheet">
      <string notr="true">QComboBox {
    background-color: #fffdf5;
    color: #5c4033;
    border: 1px solid #d2b48c;
    border-radius: 6px;
    padding: 4px 6px;
    font-size: 14px;
}

QComboBox:focus {
    border: 1px solid #a07d5e;
    background-color: #fff8e1;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 24px;
    border-left: 1px solid #d2b48c;
    background-color: #f9f3e5;
    border-top-right-radius: 6px;
    border-bottom-right-radius: 6px;
}
QComboBox::down-arrow {
    width: 12px;
    height: 12px;
    image: none;
}

</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
